composer.json

```diff
@@ -17,6 +17,10 @@
   "require": {
     "php": ">=5.4.0",
     "symfony/console": "~2.0",
-    "guzzlehttp/guzzle": "~4.0"
+        "guzzlehttp/guzzle": "~4.0",
```

Comment:
But it isn' present anymore in here: https://raw.githubusercontent.com/harikt/wireshell/installer/wireshell